mysql-test/suite/rocksdb/t/delete.test

```diff
@@ -7,9 +7,10 @@
 # DELETE IGNORE is covered in delete_ignore test
 #
 
---disable_warnings
-DROP TABLE IF EXISTS t1,t2;
---enable_warnings
+# MyRocks does not support gap locks in REPEATABLE-READ mode, test does not
+# require RR ISO to complete, so lets temporarily alter the ISO to RC
+set @orig_tx_iso=@@session.tx_isolation;
```

Comment:
True, end of tests really don't necessarily need to reset sessions, but, IIRC, I have been requested to track and reset sessions in reviews before. Perhaps it was not a realization that a set/reset was session only or that maybe future expectation is that even session sight span test cases. I'll remove them and yell at the next reviewer that asks that I reset a session var to its original value.